package.json

```diff
@@ -10,9 +10,11 @@
     "test": "jest tests"
   },
   "dependencies": {
+    "@types/webpack-env": "^1.18.1",
```

Comment:
why?

Reply:
Dunno, works without it so i removed it.